wireless_modules_py/wind_module/main.py

```diff
@@ -0,0 +1,39 @@
+import sys
+sys.path.insert(1, '../sensors')
```

Comment:
If you can find a better way to do this that would be good, but if not then leave it as is.

Reply:
I tried structuring the folders as packages by adding an `__init__.py` in `sensors` and `wireless_module_py` so that we could import like how we normally would but had no luck. As a result, had to use the `sys` module and modify the search path to import `WindSensor` and `WirelessModule`.

Reply:
That's fine continue using `sys`, it works so there's no harm in using it. If we can find a better method to use in the future we can just change the code.